youtube_dl/extractor/vidello.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import (
+    clean_html
```

Comment:
Depending on later suggestions:
```suggestion
    clean_html,
    int_or_none,
    try_get,
    url_or_none,
```